.github/workflows/release-21_extrinsic-ordering-check-from-two.yml

```diff
@@ -0,0 +1,96 @@
+# This workflow performs the Extrinsic Ordering Check on demand using a binary
+
+name: Release - Extrinsic Ordering Check against alternative binary
+on:
+  workflow_dispatch:
+    inputs:
+      reference_binary_url:
+        description: A url to a Linux binary for the node containing the reference runtime to test against
+        default: https://releases.parity.io/polkadot/x86_64-debian:stretch/v0.9.26/polkadot
+        required: true
+      binary_url:
+        description: A url to a Linux binary for the node containing the runtime to test
+        default: https://releases.parity.io/polkadot/x86_64-debian:stretch/v0.9.27-rc1/polkadot
+        required: true
+      chain:
+        description: The name of the chain under test. Usually, you would pass a local chain
+        default: kusama-local
+        required: true
+
+jobs:
+  check:
+    name: Run check
+    runs-on: ubuntu-latest
+    env:
+      CHAIN: ${{github.event.inputs.chain}}
+      BIN_URL: ${{github.event.inputs.binary_url}}
+      REF_URL: ${{github.event.inputs.reference_binary_url}}
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Fetch reference binary
+        run: |
+          echo Fetching $REF_URL
+          wget $REF_URL
+          mv polkadot polkadot-ref
```

Comment:
How about to use `wget $REF_URL -O polkadot-ref` directly and save some extra command here?